netebpfext/user/netebpfext_platform.h

```diff
@@ -15,6 +15,11 @@
 #include <netiodef.h>
 #include <../km/netioddk.h>
 
-#include "fw_thunk.h"
 #include "kernel_thunk.h"
 #include "ndis_thunk.h"
+
+// Note: fwpsk.h and fwpmk.h depend on NDIS definitions.
+// The real NDIS definitions collide with Win32 headers and hence we need to first
+// include the "ndis_thunk.h" before including the kernel mode FW headers.
```

Comment:
if "ndis_thunk.h" were renamed to just "ndis.h" then the order would not matter, one could delete line 19, and just use the include path order to make it find the correct ndis.h.